src/mod.rs

```diff
@@ -1,54 +0,0 @@
-// Copyright 2019-2023 ChainSafe Systems
-// SPDX-License-Identifier: Apache-2.0, MIT
-
-//! This module contains code that is shared between the library and the build script
```

Comment:
I'm curious about this as it does not seem like `build.rs` has any references to that.

Reply:
It used to do, but not anymore, and that's why I remove it in this PR